### ISSUE TITLE
Username was not found in the database.

### ISSUE THREAD

**Issue by Hiltos** [overhold-org/Overhold-Wallet#1]
I'm running Overhold 1.0.0 on Windows 10.
I set up a new account. That went fine. I explored the UI and closed the program.

When I opened the wallet again I entered my username and password and got an error:
"Username was not found in the database."

So I recovered the new account with the mnemonic phrase, went to the setting menu and changed my username and password back to what I had before (wrote it down this time to be sure.)

But once again after closing and opening the wallet and trying my username and password, I still get the error. It's like my username and/or password are not being saved.

**Comment by blag0j** [comment]
@Hiltos, thanks for the explanation, we're trying to reproduce the problem. Could you give us some more information if you're running the program as administrator, and also are you trying to run the main bundled executable, or the executable from the unpacked version?

Please try to run the program as administrator (if you are not doing so), and let us know the result.
Thanks.

**Comment by Hiltos** [comment]
Oh I see what my problem was. I didn't realize that there was an executable in the unpacked folder. I was always running the main bundled executable. I got it to work now without issues. Here's what I did.

I ran the executable in the win-unpacked folder as an administrator. At this point it wasn't accepting any username/password that I tried. So I entered my mnemonic to recover my account. This time I got a warning saying that there was already an account registered and this would overwrite it. I was fine with that because I've only ever had a new account anyway.

So then I entered the reset credentials (Username = username, Password = password) which let me in. I managed to enter my own username and password in the settings menu. No problems there. I was then able to sign out and log back in using my new username and password.

So there's really no bug here. I was just using the wrong executable.

Thanks! Hope this helps.
